web/src/stream_create_subscribers.js

```diff
@@ -97,6 +97,7 @@ export function build_widgets() {
                 full_name: user.full_name,
                 is_current_user: user.user_id === current_user_id,
                 disabled: stream_create_subscribers_data.must_be_subscribed(user.user_id),
+                src: user.avatar_url,
```

Comment:
I cannot see avatar for bots and there can be case where `avatar_url` field is not defined. It is better to use `people.small_avatar_url_for_person` here. Also, I think we could have a better name for this variable like `img_src`.

Reply:
I have replace ```src``` to ```img_src``` and ```people.small_avatar_url_for_person``` instead of ```user.avatar_url``` locally.